lib/openstax/salesforce/spec_helpers/salesforce_proxy.rb

```diff
@@ -114,7 +114,7 @@ def school_id(name)
     end
 
     def school(name)
-      schools.select{|ss| ss.name == name}.first
+      schools.find { |ss| ss.name == name }
     end
```

Comment:
Hah!  I actually ran blame because I was afraid I'd done the `select..first` thing, but nope good old @jpslav was the culprit  😁